### PR TITLE
Small English translation fixes.

### PR DESCRIPTION
**Pull request by lseelenbinder**

### CODE REVIEW

File: app/locales/en.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
   "Join us in the most important crowdsource!": "Join us in the most important crowdsource!",
   "Make your voluntary contribution to the health care system and register your health condition now. Together we can create an overview.": "Make your voluntary contribution to the health care system and register your health condition now. Together we can create an overview.",
   "Share on": "Share on",
-  "NB!": "NB!",
+  "NB!": "Important!",
```
Comment (michaelmcmillan) [inline app/locales/en.json:50]: Should we have a trailing colon here?

File: app/locales/en.json [hunk 2]
```diff
@@ -105,17 +105,16 @@
   "Submit report": "Submit report",
   "Statistics": "Statistics",
   "Download the dataset as CSV": "Download the dataset as CSV",
-  "COVID-19 infected": "COVID-19 infected",
-  "people have reported that they have tested positive for COVID-19 and experience symptoms.": "people have reported that they have tested positive for COVID-19 and experience symptoms.",
+  "COVID-19 infected": "COVID-19 infected", "people have reported that they have tested positive for COVID-19 and experience symptoms.": "people have reported that they have tested positive for COVID-19 and experienced symptoms.",
```
Comment (michaelmcmillan) [inline app/locales/en.json]: Looks like this line is missing a linebreak (two translations on one line)

**Review by michaelmcmillan — CHANGES_REQUESTED** [review]
Nice work! Added some nitpick

File: app/locales/en.json (hunk 1 above)
Comment (lseelenbinder) [reply app/locales/en.json:50]: Either a trailing colon or the exclamation point, as-is, but not both.

File: app/locales/en.json (hunk 2 above)
Comment (lseelenbinder) [reply app/locales/en.json]: Oh, whoops! Good catch.

File: app/locales/en.json (hunk 1 above)
Comment (michaelmcmillan) [reply app/locales/en.json:50]: Ah my bad, I misread, exclamation point should do!

File: app/locales/en.json (hunk 2 above)
Comment (lseelenbinder) [reply app/locales/en.json]: Fixed.

**Review by michaelmcmillan — APPROVED** [review]

**Comment by Kilian** [comment]
:tada: thanks!

**Pull request merged**